app/views/recipes/_recipe.html.erb

```diff
@@ -0,0 +1,56 @@
+<div 
+  id="<%= dom_id recipe %>"
+  <% if recipe_show %>
+    <% recipeCardClass = "recipe-card recipe-card-show" %>
+  <% else %>
+    <% recipeCardClass = "recipe-card" %>
+  <% end %>
+  class="<%= recipeCardClass %>"
+>
+  <div class="recipe-card-top">
+    <h2>
+      <%= recipe.name %>
+    </h2>
+    <% if recipe_index %>
+
+      <%= button_to "Delete", user_recipe_path(current_user, recipe), method: :delete, data: { confirm: "Are you sure?" }, class:"black-btn delete-recipe" %>
```

Comment:
To stick to the project requirement and though it is not that much important, kindly change the `Delete` button to `Remove` button.

Reply:
Thanks @amare1990 for reviewing my code. 👋🏻

I'll consider this change for the next stages of the project.